### ISSUE TITLE
TCP Aliases support 

### ISSUE THREAD

**Issue by dpishchukhin** [antoniomika/sish#35]
Hello @antoniomika,

Can you add TCP aliases to your great tool? I found this functionality by another tool.
Here are examples to explain what I want:

- Create a TCP tunnel with alias: `ssh -p 2222 -R myalias:22:localhost:22 ssi.sh` 
- Connect to the tunnel: `ssh -o ProxyCommand="ssh -W myalias:22 -p 2222 ssi.sh" user@myalias`

I see a lot of advantages to this approach:
- use the same port to create TCP tunnels
- TCP tunnels could be named like subdomains for HTTP(S)

**Comment by antoniomika** [comment]
I see the use case, but is there any security concerns that should be worried about when a user set's up an alias? I guess since this is self hosted that wouldn't be as large of an issue. i will take a look though!

**Comment by antoniomika** [comment]
Hey @dpishchukhin!

I just merged the PR that contains support for TCP aliases. They work more or less exactly how you listed (I tested it using an SSH proxy command also). The only thing to note is an alias is considered the pair of port and hostname (i.e. the alias is myalias:22, not just myalias). 

The reason for this was to 
1. Provide less ability for enumeration, and
2. Allow for reusing an alias name (think hostname) for a single SSH command with multiple forward requests. That is, something like: 
    - ```ssh -p 2222 ssi.sh -R $(hostname):22:localhost:22 R $(hostname):8443:localhost:8443``` can be used.
